### DISCUSSION TITLE
Dont Auto-Extract file

### DISCUSSION THREAD

**Discussion by Vicenzo-Giuseppe** [ScoopInstaller/Scoop#5732] [Ideas]
i think its make easier to use the .tar.gz because wsl works directly with .tar.gz and if i let extract the program could not have the permissions to extract and i tested in explorer and for me too its impossible too to extract manually.
[nixos-wsl.json](https://github.com/ScoopInstaller/Scoop/files/13402417/nixos-wsl.json)

**Comment by HUMORCE** [comment]
```
url: https://.../nixos-wsl-x86_64-linux.tar.gz#/dl.tar.gz_
pre_install: Rename-Item $dir\$fname -NewName 'nixos-wsl-x86_64-linux.tar.gz'
...
```

- https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests#optional-properties : Part of url
- https://github.com/ScoopInstaller/Scoop/wiki/Pre-Post-(un)install-scripts
